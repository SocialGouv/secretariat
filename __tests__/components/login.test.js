import { render } from "@testing-library/react"

import Login from "../../src/components/login"

it("renders homepage", () => {
  const { container } = render(<Login />)
  expect(container).toMatchSnapshot()
})
