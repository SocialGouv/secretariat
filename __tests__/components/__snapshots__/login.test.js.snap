// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login 1`] = `
<div>
  <div
    class="login"
  >
    Pour vous connecter, cliquez sur le bouton juste en dessous, l√†: 
    <br />
     
    <br />
    <button
      class="primary"
    >
      Connexion
    </button>
  </div>
</div>
`;
